sql/PHQ9 final 021517.sql

```diff
@@ -0,0 +1,180 @@
+
+--
+-- Dumping data for table `list_options`
+--
+-- WHERE:  list_id = 'lists' AND option_id = 'PHQ9_Severity' OR list_id = 'PHQ9_Severity'
+
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('lists','PHQ9_Severity','PHQ9 Severity',301,1,0,'','','',0,0,1,'');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('PHQ9_Severity','1','0-4: None',10,0,0,'','','',0,0,1,'');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('PHQ9_Severity','2','5-9: Mild',20,0,0,'','','',0,0,1,'');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('PHQ9_Severity','3','10-14: Moderate',30,0,0,'','','',0,0,1,'');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('PHQ9_Severity','4','15-19: Moderately Severe',40,0,0,'','','',0,0,1,'');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('PHQ9_Severity','5','20-27: Severe',50,0,0,'','','',0,0,1,'');
+/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;
+
+/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
+/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
+/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;
+
+-- Dump completed on 2017-02-16 11:12:08
+DELETE FROM layout_options WHERE form_id = 'LBFPHQ9';
+DELETE FROM list_options WHERE list_id = 'lbfnames' AND option_id = 'LBFPHQ9';
+-- MySQL dump 10.14  Distrib 5.5.50-MariaDB, for Linux (x86_64)
+--
+-- Host: localhost    Database: openemr
+-- ------------------------------------------------------
+-- Server version	5.5.50-MariaDB
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Dumping data for table `list_options`
+--
+-- WHERE:  list_id = 'lbfnames' AND option_id = 'LBFPHQ9'
+
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`) VALUES ('lbfnames','LBFPHQ9','PHQ9 Questionnaire',30,0,0,'','','',0,0,1,'');
```

Comment:
Looks like you have a nonstandard number of columns in the layout? Is that set anywhere?